### PR TITLE
Added limitations to docs: app development and Spark shell

### PR DESCRIPTION
**Pull request by susanxhuynh**
Added documentation about not supporting Spark app development and Spark shell needing to run inside cluster.

### CODE REVIEW

**Comment by susanxhuynh** [comment]
@mgummelt @sascala Please review. Thanks.

File: docs/limitations.md [hunk 1]
```diff
@@ -8,10 +8,13 @@ enterprise: 'yes'
...
 does not support R.
 
+*   Mesosphere does not provide support for Spark app development.
+
 *   Spark jobs run in Docker containers. The first time you run a
 Spark job on a node, it might take longer than you expect because of
 the `docker pull`.
 
-*   For interactive analytics, we
+*   DC/OS Spark only supports running the Spark Shell from within the
```
Comment (mgummelt) [inline docs/limitations.md]: Can you provide a link to these instructions?

**Comment by mgummelt** [comment]
One comment then LGTM

File: docs/limitations.md (hunk 1 above)
Comment (sascala) [inline docs/limitations.md]: suggest "DC/OS Spark only supports running the Spark shell from within a DC/OS cluster. See the Spark Shell section for more information."

**Comment by susanxhuynh** [comment]
@keithchambers Alright, I will create a separate PR for that.

**Pull request merged**